src/status_im/contexts/wallet/share_community_channel/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im.contexts.wallet.share-community-channel.view
```

Comment:
oh please move this ns to 
`status-im.contexts.community.actions.share-community-channel.view`